eras/conway/impl/cardano-ledger-conway.cabal

```
@@ -75,14 +75,14 @@ library
         aeson >=2.2,
         data-default-class,
         cardano-crypto-class,
-        cardano-data >=1.1.2.0,
-        cardano-ledger-binary >=1.2,
-        cardano-ledger-allegra >=1.1,
+        cardano-data ^>=1.2,
```

No speculative upper bounds
```suggestion
        cardano-data >=1.2,
```